session/test_kvdb.go

```diff
@@ -17,7 +18,24 @@ func NewTestDB(t *testing.T, clock clock.Clock) *BoltStore {
 func NewTestDBFromPath(t *testing.T, dbPath string,
 	clock clock.Clock) *BoltStore {
 
-	store, err := NewDB(dbPath, DBFilename, clock)
+	acctStore := accounts.NewTestDB(t, clock)
+
+	store, err := NewDB(dbPath, DBFilename, clock, acctStore)
+	require.NoError(t, err)
+
+	t.Cleanup(func() {
+		require.NoError(t, store.DB.Close())
+	})
+
+	return store
+}
+
+// NewTestDBWithAccounts creates a new test session Store with access to an
+// existing accounts DB.
+func NewTestDBWithAccounts(t *testing.T, clock clock.Clock,
+	acctStore accounts.Store) *BoltStore {
+
+	store, err := NewDB(t.TempDir(), DBFilename, clock, acctStore)
```

Comment:
Nit:

Could potentially create a helper function that both of these use instead of both functions repeating the:
```
	store, err := NewDB(t.TempDir(), DBFilename, clock, acctStore)
	require.NoError(t, err)

	t.Cleanup(func() {
		require.NoError(t, store.DB.Close())
	})

	return store
```
part with slightly different args.

Reply:
sure